### PR TITLE
Fixes #151 - Separate console app from web application

### LINKED ISSUE #151: Enable KInspector.Web project to run in IIS
The KInspector.Web project is a self-hosted .NET Core application, but is not currently compatible with IIS. By enabling this to run in IIS without the accompanying console window will enable easier use in remote scenarios and on developer machines.

### PR DESCRIPTION
**Pull request by ChristopherJennings**

### CODE REVIEW

**Comment by ChristopherJennings** [comment]
Note: This is a pull request into the v4-dev branch, not master.

**Comment by ChristopherJennings** [comment]
FYI AppVeyor build failed because of roslyn compiler nuget restore issue. I updated the build to do a buget restore before build in order to resolve.

File: KInspector.Core/DatabaseService.cs [hunk 1]
```diff
@@ -14,6 +15,7 @@ public class DatabaseService : IDatabaseService
     {
         private readonly string mConnectionString;
         private const int SQL_COMMAND_TIMEOUT_SECONDS = 90;
+        private static string executingPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().CodeBase.Substring(8));
```
Comment (petrsvihlik) [inline KInspector.Core/DatabaseService.cs]: no magic constants pls. at least do something like `"abcdefgh".Length`

File: KInspector.Core/DatabaseService.cs [hunk 2]
```diff
@@ -96,7 +98,7 @@ private DataTable ExecuteAndGetTable(string sql, params SqlParameter[] parameter
         /// <param name="parameters">Optional parameters send to SQL script</param>
         public DataTable ExecuteAndGetTableFromFile(string filePath, params SqlParameter[] parameters)
         {
-            using (var sr = new StreamReader("./Scripts/" + filePath))
+            using (var sr = new StreamReader( executingPath + "\\Scripts\\" + filePath))
```
Comment (petrsvihlik) [inline KInspector.Core/DatabaseService.cs]: define a "scriptPath" field...it's repeated on three lines

File: KInspector.Core/ModuleLoader.cs [hunk 3]
```diff
@@ -30,13 +32,13 @@ public static ICollection<IModule> Modules
                 return mModules.Values;
             }
         }
-
-
+        
         private static void LoadModules()
         {
+            var path = Path.GetDirectoryName(Assembly.GetExecutingAssembly().CodeBase.Substring(8));
```
Comment (petrsvihlik) [inline KInspector.Core/ModuleLoader.cs]: ...

File: KInspector.Modules/Export/ExportModuleLoader.cs [hunk 4]
```diff
@@ -35,9 +37,10 @@ public static ICollection<IExportModule> Modules
 
         private static void LoadModules()
         {
+            var path = Path.GetDirectoryName(Assembly.GetExecutingAssembly().CodeBase.Substring(8));
```
Comment (petrsvihlik) [inline KInspector.Modules/Export/ExportModuleLoader.cs]: ...

**Comment by petrsvihlik** [comment]
hey @ChristopherJennings 
could you please take care of this PR? it's been here for four months...

**Pull request merged**